commands/channel_test.go

```diff
@@ -2176,3 +2176,268 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestCreateChannelCmd() {
+	s.Run("should not create channel without display name", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		teamName := "teamName"
+		channelName := "channelName"
+		argsTeamChannel := teamName + ":" + channelName
+		args := []string{""}
+		args[0] = argsTeamChannel
+
+		cmd.Flags().String("team", teamName, "Team Name")
+		cmd.Flags().String("name", channelName, "Channel Name")
+
+		err := createChannelCmdF(s.client, cmd, args)
+		s.Require().EqualError(err, fmt.Sprintf("display Name is required"))
+	})
+
+	s.Run("should not create channel without name", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		teamName := "teamName"
+		channelDisplayName := "channelDisplayName"
+		argsTeamChannel := teamName + ":" + channelDisplayName
+		args := []string{""}
+		args[0] = argsTeamChannel
+
+		cmd.Flags().String("team", teamName, "Team Name")
+		cmd.Flags().String("display_name", channelDisplayName, "Channel Display Name")
+
+		err := createChannelCmdF(s.client, cmd, args)
+		s.Require().EqualError(err, fmt.Sprintf("name is required"))
+	})
+
+	s.Run("should not create channel without team", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		channelName := "channelName"
+		channelDisplayName := "channelDisplayName"
+		argsTeamChannel := channelName + ":" + channelDisplayName
+		args := []string{""}
+		args[0] = argsTeamChannel
+
+		cmd.Flags().String("name", channelName, "Channel Name")
+		cmd.Flags().String("display_name", channelDisplayName, "Channel Display Name")
+
+		err := createChannelCmdF(s.client, cmd, args)
+		s.Require().EqualError(err, fmt.Sprintf("team is required"))
+	})
+
+	s.Run("should fail when team does not exist", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+
+		teamName := "teamName"
+		channelName := "channelName"
+		channelDisplayName := "channelDisplayName"
+		argsTeamChannel := teamName + ":" + channelName + ":" + channelDisplayName
+		args := []string{""}
+		args[0] = argsTeamChannel
+
+		cmd.Flags().String("team", teamName, "Team Name")
+		cmd.Flags().String("name", channelName, "Channel Name")
+		cmd.Flags().String("display_name", channelDisplayName, "Channel Display Name")
+
+		s.client.
+			EXPECT().
+			GetTeam(teamName, "").
+			Return(nil, &model.Response{Error: nil}).
```

Comment:
nit: This should be returning an error. Even though we don't check the error, we should add some mock error here.